change/@itwin-measure-tools-react-94a7f921-b7a3-4caf-b3e2-9ab99a2fafb6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Measurement in sheets",
+  "packageName": "@itwin/measure-tools-react",
+  "email": "maxime.brassard@bentley.com",
```

Comment:
You can use @bentley, but ideally use the no-reply email from GH to avoid being spammed with undesirable stuff since this is public source

Reply:
RIP his email folder